config.yaml

```diff
@@ -0,0 +1,43 @@
+facility: sandbox
+log_level: debug
+namespace: default
+store_kind: fleetdb
```

Comment:
I feel like this is a leftover for when we were developing things in the beginning and we should look at what it would take to retire `store_kind` as a concept.

Reply:
im down to gut the idea.

Reply:
dont mind if i do